Formula/lima.rb

```diff
@@ -0,0 +1,31 @@
+class Lima < Formula
+  desc "Linux-on-Mac - macOS subsystem for Linux - containerd for Mac"
```

Comment:
`desc “Linux virtual machines, on macOS (aka \"Linux-on-Mac\", \"macOS subsystem for Linux\", \"containerd for Mac\", unofficially)”`

Reply:
Fine for me, will update that

Reply:
Could we have something that's shorter? I don't think having all those alternate names is helpful. Instead we should describe the Formula for what its functionality is, succinctly.